addons/@mlabs-haskell/godot-cardano/bin/libcsl_godot.gdextension

```diff
@@ -21,3 +21,5 @@ android.debug.x86_64 = "res://addons/@mlabs-haskell/godot-cardano/bin/libcsl_god
 android.release.x86_64 = "res://addons/@mlabs-haskell/godot-cardano/bin/libcsl_godot.android.template_release.x86_64.so"
 android.debug.arm64 = "res://addons/@mlabs-haskell/godot-cardano/bin/libcsl_godot.android.template_debug.arm64.so"
 android.release.arm64 = "res://addons/@mlabs-haskell/godot-cardano/bin/libcsl_godot.android.template_release.arm64.so"
+web.debug.wasm32 = "res://addons/@mlabs-haskell/godot-cardano/bin/csl_godot.wasm"
```

Comment:
I'm not sure atm what is the proper way to rename the `.wasm` file. It looks like, it will fail do load library in web-export if file name does not match `name` in `config.toml`. Maybe changing filename [in HTML shell config](https://github.com/mlabs-haskell/godot-cardano/pull/67/files#diff-fe73f6ab0a32d480975bee9889c280ec22016e1010d849b08c97d712154bcae1R225) will be enough - need to test.

Ma